README.md

```diff
@@ -75,25 +87,46 @@ OPTIONS:
   -h, --help              Show help information.
 ```
 
-## Documentation
+</details>
+
+In addition to the options passed to the command, you can also define a configuration file to customize the output in various different ways. For a complete list of options, check out the [Configuration Options](./Docs/ConfigOptions.md) documentation.
 
-- [Configuration](./Docs/ConfigOptions.md)
+For more information about using CreateAPI, check out the [Documentation](./Docs/).
 
-## OpenAPI Support
+## Projects using CreateAPI
 
-The goal is to completely cover OpenAPI 3.x spec.
+Need some inspiration? Check out the list of projects below that are already using CreateAPI:
 
-Currently, the following features are **not** supported:
+- [appstoreconnect-swift-sdk](https://github.com/AvdLee/appstoreconnect-swift-sdk)
 
-- External References
+Are you using CreateAPI in your own open-source project? Let us know by [adding it](https://github.com/CreateAPI/CreateAPI/edit/main/README.md) to the list above!
 
-Some discrepancies with the OpenAPI spec are by design:
+## Contributing
 
-- `allowReserved` keyword in parameters is ignored and all parameter values are percent-encoded
-- `allowEmptyValue` keyword in parameters is ignored as it's not recommended to be used
+We always welcome contributions from the community via Issues and Pull Requests. Please be sure to read over the [contributing guidelines](./CONTRIBUTING.md) for more information.
 
-Upcoming:
+## Releasing
 
-- An improved way to generate patch parameters. Support for [JSON Patch](http://jsonpatch.com).
-- OpenAPI 3.1 support.
+When releasing a new version of CreateAPI, be sure to follow the steps outlined below:
 
+- [ ] Ensure that the `main` branch checks are passing.
+- [ ] Update the [version number](https://github.com/CreateAPI/CreateAPI/blob/main/Sources/CreateAPI/CreateAPI.swift#L8).
+- [ ] Update the `create-api generate --help` output in **README.md** if it has changed.
+- [ ] Update [CHANGELOG.md](./CHANGELOG.md).
+  - Use the GitHub generated release notes as a base. Be sure to cleanup the PR links.
+- [ ] Create a GitHub Release.
+  - Create a new tag using semantic versioning.
+  - Use the generated release notes.
+- [ ] From the [Release Workflow](https://github.com/CreateAPI/CreateAPI/actions/workflows/release.yml), get the artifactbundle SHA and add an Artifact Bundle section to the release page. For example:
+    > ## Artifact Bundle
+    >
+    > Checksum: `89c75ec3b2938d08b961b94e70e6dd6fa0ff52a90037304d41718cd5fb58bd24`
+    >
+    > ```swift
+    > .binaryTarget(
+    >     name: "create-api",
+    >     url: "https://github.com/CreateAPI/CreateAPI/releases/download/0.0.5/create-api.artifactbundle.zip",
+    >     checksum: "89c75ec3b2938d08b961b94e70e6dd6fa0ff52a90037304d41718cd5fb58bd24"
+    > )
+    > ```
+- [ ] Push the release to Homebrew with `brew bump-formula-pr create-api`.
```

Comment:
I had been meaning to note this down somewhere, maybe README.md isn't the best place for it though?

Reply:
Agree that maybe the README isn't the best fit for this. I thought about suggesting you move it to `CONTRIBUTING.md`, but maybe a better option is a new file entirely. You could still link to it from the readme, or from contributing.

Reply:
I think linked from contributing would be the way, especially since releases won't be made unless you have access to.

Reply:
Great idea, I'll move it to **CONTRIBUTING.md**, thanks